### ISSUE TITLE
a11y: VS Code intermittently inaccessible to screen reader after upgrade to 1.69.0

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#3398]
Immediately after upgrading to VS Code 1.69.0 on Windows 10, it intermittently stopped working with my screen reader. More specifically, starting VS Code usually doesn't present anything to NVDA. While I can OCR the window, I can't tab through or interact with controls and have them spoken. No other changes were made when this behavior started.

Sometimes it works, sometimes it doesn't. I haven't narrowed down a specific set of circumstances that makes it work vs. not. Sometimes it works/doesn't work with the exact same project. Restarting NVDA with the project loaded doesn't get it working.

I'm certain the project is loaded. It's Rust, so Rust Analyzer chugs away for several minutes. But the window remains inaccessible even after RA finishes its churn. And again, OCRing shows it working.

I've restarted NVDA and my PC multiple times.

This can't continue. Where can I get VS Code 1.68, and if I install it, how do I keep it from auto-upgrading to this latest version? And are there any other details I can provide? Hopefully someone else can duplicate this.
